apps/docs/pages/guides/auth/phone-login/twilio.mdx

```diff
@@ -13,21 +13,49 @@ In this guide we'll show you how to authenticate your users with SMS based One-T
 
 There are two reasons to use Supabase SMS OTP tokens:
 
-- You want users to log in with mobile number + password, and the mobile number should be verified via SMS
-- You want users to log in with mobile number ONLY (i.e. passwordless login)
-
-We'll cover:
-
-- [Finding your Twilio credentials](#finding-your-twilio-credentials)
-- [Using OTP with password based logins](#using-otp-with-password-based-logins)
-- [Using OTP as a passwordless sign-in mechanism](#using-otp-as-a-passwordless-sign-in-mechanism)
+- You want users to log in with a phone number and password, and verify the phone number on signup with SMS
+- You want users to log in with a phone number ONLY (i.e. passwordless login)
 
 What you'll need:
 
 - A Twilio account (sign up here: https://www.twilio.com/try-twilio)
 - A Supabase project (create one here: https://app.supabase.com)
 - A mobile phone capable of receiving SMS
 
+SMS Authentication can be done with either Twilio (Programmable Messaging) or Twilio Verify. [Twilio Verify](https://www.twilio.com/en-us/trusted-activation/verify) is a service that replaces [Twilio Programmable Messaging](https://www.twilio.com/docs/messaging). Twilio Verify is a specialized OTP solution and is recommended for developers requiring features like fraud protection.
+
+<Tabs>
+<TabPanel id="twilio-verify" label="Twilio Verify">
```

Comment:
Why tabs?

Reply:
Maybe just get rid of tabs. just have some buttons at top to jump to correct part of the page.

Reply:
also avoids any SEO crawling issues

Reply:
Initially didn't want to push too hard on one solution so thought that using tabs would present them as alternatives but with Twilio as the slightly preferred alternative since it's the one displayed.

Removing the tabs